lib/fs.js

```diff
@@ -597,9 +611,23 @@ function readSync(fd, buffer, offset, length, position) {
 
   validateOffsetLengthRead(offset, length, buffer.byteLength);
 
-  if (!NumberIsSafeInteger(position))
+  if (position == null)
     position = -1;
```

Comment:
It could the optimized to have a `else`? Once that `if (position == null)`, then `position = -1`, so it is a valid integer and not need be checked by the next ìf (typeof position === 'number')`, right?

```js
if (position == null) {
    position = -1;
}
else if (typeof position === 'number') {
    // ...
}
```

Reply:
Sure, that would work. Do you want to open a PR with this change?

Reply:
@aduh95 if possible, you can do it?

Reply:
I was actually thinking about doing what was discussed here: https://github.com/nodejs/node/pull/36190#discussion_r540160570
That way, `position` does not need to be set to `-1` if it has a `null` value.

Reply:
@rentalhost see https://github.com/nodejs/node/pull/37101.